cli/cmd/draft.go

```diff
@@ -44,7 +45,7 @@ func init() {
 
 var draftCmd = &cobra.Command{
 	Use:   "draft",
-	Short: "Creates Humanitec deployment draft from the source SCORE file",
```

Comment:
We now have a lot of code duplication.
May we invoke `delta(..)` function for the `draft` command instead?
```
var draftCmd = &cobra.Command{
	Use:   "draft",
	Short: "DEPRECATED - use 'delta' instead - creates Humanitec deployment draft from the source SCORE file",
	RunE:  delta,
}
```

Reply:
Sure! Code duplication removed in next commit.